api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -32,11 +32,7 @@
 import org.pmiops.workbench.cohortreview.ReviewQueryBuilder;
 import org.pmiops.workbench.cohortreview.util.ParticipantCohortStatusDbInfo;
 import org.pmiops.workbench.db.dao.UserRecentResourceService;
-import org.pmiops.workbench.db.model.Cohort;
-import org.pmiops.workbench.db.model.CohortReview;
-import org.pmiops.workbench.db.model.ParticipantCohortStatus;
-import org.pmiops.workbench.db.model.ParticipantCohortStatusKey;
-import org.pmiops.workbench.db.model.User;
+import org.pmiops.workbench.db.model.*;
```

Comment:
nit: check IJ settings. Some folks are militant about not importing *, but it's actually an area I don't have an opinion on.

Reply:
Looks like we don't have a convention on this, so star away.

Reply:
There's a team convention around not using `*` imports. However, this wasn't written down anywhere so arguably it's not official.

Reply:
I propose adding it to the Spotless rules and/or IntelliJ settings file if we want to make it a norm.

Reply:
Ugh I thought I already got IntelliJ to stop doing this